### DISCUSSION TITLE
The screen becomes blurred when IO expander is enabled.

### DISCUSSION THREAD

**Discussion by prowong** [davetcc/tcMenu#344] [Q&A]
**Describe the bug**
The tcmenu works well until the PCF8575 IO expander is selected in the tcmenu desiginer. generate and upload the code, the screen becomes blurred, event  I haven't wired them up yet.

**Commerical or Personal use**
 Personal  use

**Embedded environment**
Arduino ide 1.8.19
Board: ESP8266

Display: ST7789 driven by TFT_eSPI, connected with HW SPI
TFT_CS----GPIO 15
TFT_DC----GPIO 0
TFT_RST----GPIO 2

PCF8575 expander board, address: 0x20
SCL----GPIO 5
SDA----GPIO 4

**Screenshots**
![IMG_20230501_180157-50%-25%](https://user-images.githubusercontent.com/8499801/235444994-8cfbe7b4-35d1-447c-aca3-249544c1110f.jpg)

Before adding the IO expander, it looks normal.

![IMG_20230501_190209-25%-50%](https://user-images.githubusercontent.com/8499801/235445015-8ef97fa6-fabd-403c-b082-b5dbb4ddae49.jpg)

Is my setting wrong? Your projects are very helpful for the beginner like me, thank you.

**Comment by davetcc** [comment]
Please raise this as a discussion, it is not an issue. Issues are confirmed broken behavior in the libraries somewhere, this is most likely an electrical issue on your board.

**Reply by davetcc** [reply]
I've moved this to a discussion, have you checked for electrical interference, it is highly unlikely that either TFT_eSPI or tcMenu libraries caused this. 

Start with the simplest TFT_eSPI example and check if it exhibits the same symptoms first. With the simplest graphical example running you can then try to evaluate if you are taking too much power from the board or have wiring issues.

**Comment by prowong** [comment]
Sorry it was my mistake. I finally realized that the DC pin cannot be connected to GPIO0.  After I reconfigured the gpio it worked fine. Confusingly it runs the graphical example or the menu example appears to be displayed normally. Anyway thank you very much.
